backend/fms_core/resources/sample.py

```diff
@@ -178,16 +178,15 @@ def import_obj(self, obj, data, dry_run):
             errors["individual"] = ValidationError("Individual could not be processed due to missing or invalid information.", code="invalid")
 
         #Individual field errors
-        validate_specific_field(errors, "Taxon", "taxon", data)
-
-        validate_specific_field(errors, "Individual ID", "individual_id", data)
-
-        #Container field errors
-        validate_specific_field(errors, "Container Kind", "container_kind", data)
-
-        validate_specific_field(errors, "Container Name", "container_name", data)
-
-        validate_specific_field(errors, "Container Barcode", "container_barcode", data)
+        fields = {
```

Comment:
a suggestion regarding the variable names; fields is a very broad variable name. could be difficult to search through the codebase with a Ctrl + F. I would suggest something more specific and explicit such as attribute_by_field_name or attribute_with_field_name, or field_name_attribute_dict; so anything that hints that the structure is a map/dictionary, and not just a list.

Reply:
Perfect!